components/molecules/InsightHeader/insight-header.tsx

```diff
@@ -8,6 +11,8 @@ import ContextThumbnail from "components/atoms/ContextThumbnail/context-thumbnai
 import CardRepoList from "../CardRepoList/card-repo-list";
 import { truncateString } from "lib/utils/truncate-string";
 import useRepositories from "lib/hooks/api/useRepositories";
+import { useEffect, useState } from "react";
```

Comment:
nit: move to top of file

```suggestion
import { useEffect, useState } from "react";
```